homeassistant/components/ecovacs/vacuum.py

```diff
@@ -56,14 +63,13 @@ def __init__(self, device: sucks.VacBot) -> None:
         """Initialize the Ecovacs Vacuum."""
         self.device = device
         self.device.connect_and_wait_until_ready()
```

Comment:
Side note: Probably move this to the init module if it doesn't matter if it's done before the event listener subscriptions or at the end of `async_added_to_hass` if it needs to happen after the subscriptions.

Reply:
this is planned to be done, when making the integration setup async (_because it's actually blocking HA startup_)

Reply:
#96200